### PR TITLE
Skip similarity investigation if replays both have Autopilot

### LINKED ISSUE #153: Ignore aim when comparing AP plays
image

### PR DESCRIPTION
**Pull request by Stedoss**
Most likely an edge case for must users, but nice to have.

Resolves #153.

### CODE REVIEW

File: circleguard/gui/main_tab.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from PyQt6.QtWidgets import (QMessageBox, QFrame, QGridLayout,
     QApplication, QToolTip, QLabel, QSizePolicy, QTextBrowser)
 from PyQt6.QtGui import QTextCursor
+from circleguard import Mod
```
[the remark this answers is not in the document]
Comment (Stedoss) [reply circleguard/gui/main_tab.py]: That's what I get for auto-importing, nice catch

**Review by tybug — APPROVED** [review]
ty

**Comment by tphinney** [comment]
@lmck620 Please stop whatever process you are doing that involves crossposting all your comments in other repositories, to the Google Material Icons repository. I expect this is unintentional, but it is quite irksome for those of us on that repository.

**Comment by tybug** [comment]
Equally bothersome for myself, whose comments are getting caught in the crossfire and potentially misattributed 😕.

I've just blocked the user and moved on, myself.

**Pull request merged**